service/src/main/java/io/camunda/service/exception/NotFoundException.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service.exception;
+
+public class NotFoundException extends RuntimeException {
```

Comment:
🔧 If we want to create a specific exception for the services (which might make sense here), we could build this on top of https://github.com/camunda/camunda/blob/main/service/src/main/java/io/camunda/service/CamundaServiceException.java. We use that exception class to determine if the error from the service was a specific broker exception or just any arbitrary exception, for example. We can also extend the `CamundaServiceException`, of course, if we want to differentiate that this was a "search exception", for example.

Reply:
I made `NotFoundException` extending `CamundaServiceException`. It can be used in the endpoints requiring an object key to get or to mutate this object